samples/host_exerciser/host_exerciser.h

```diff
@@ -346,12 +375,14 @@ class host_exerciser : public test_afu {
     app_.add_option("--contmodetime", he_contmodetime_,
         "Continuous mode time in seconds")->default_val("0");
 
+    app_.add_option("--clock-mhz", he_clock_mhz_,
```

Comment:
Should he_clock_mhz_ be initialized in the constructor?

Reply:
Isn't that what default_val is for? That's how all the other params are handled, too.